client/state/posts/test/selectors.js

```diff
@@ -364,4 +360,30 @@ describe( 'selectors', () => {
 			expect( isRequesting ).to.be.false;
 		} );
 	} );
+
+	describe( '#getPostsBySiteIdAndPostId()', () => {
+		it( 'should return global post id by siteId and postId', () => {
```

Comment:
>If we want to add cache tests, I would suggest adding cases for a clean cache, and also other ones to test that the cache break works.

I don't think we need to test caching, as we can assume this is already working as expected via tests in `lib/create-selector/test/index.js` . My point is more that each test should be run in isolation, and we cannot be certain of that unless we reset the memoized function back to its original state between each test.